packages/pass-style/src/error.js

```diff
@@ -87,22 +87,22 @@ export const makeRepairError = () => {
     return undefined;
   }
 
-  const er1StackDesc = getOwnPropertyDescriptor(Error('er1'), 'stack');
-  const er2StackDesc = getOwnPropertyDescriptor(makeTypeError(), 'stack');
+  const typeErrorStackDesc = getOwnPropertyDescriptor(makeTypeError(), 'stack');
+  const errorStackDesc = getOwnPropertyDescriptor(Error('obligatory'), 'stack');
 
   if (
-    er1StackDesc === undefined ||
-    er2StackDesc === undefined ||
-    er1StackDesc.get === undefined
+    typeErrorStackDesc === undefined ||
+    typeErrorStackDesc.get === undefined ||
+    errorStackDesc === undefined
```

Comment:
Actually thinking of it, I would prefer to move the `errorStackDesc === undefined` into the next `if` statement.